tests/CLITests/CLITests.fs

```diff
@@ -0,0 +1,84 @@
+module CLITests
+
+open Expecto
+open Expecto.Tests
+open Expecto.Impl
+open Expecto.Logging
+open System.IO
+open System.Diagnostics
+
+open JUnit
+
+type CLIContext() =
+    static member create 
+        (
+            ?p: string,
+            ?o : string
+        ) = 
+            fun f ->
+                let tool = $"arc-validate"
+                let args = 
+                    [
+                        p |> Option.map (fun p -> $"-p {p} ") |> Option.defaultValue ""
+                        o |> Option.map (fun o -> $"-o {o} ") |> Option.defaultValue ""
+                    ]
+                    |> String.concat ""
+
+                let outFile = 
+                    o
+                    |> Option.defaultValue (System.Environment.GetEnvironmentVariable("ARC_PATH")) // default to ARC_PATH if argument is not provided
+                    |> fun s -> 
+                        if p.IsSome then 
+                            p.Value else 
+                        if System.String.IsNullOrWhiteSpace(s) then 
```

Comment:
Why no `elif`? 😂

Reply:
if that's the only thing that was unclear then i am very proud of this PR.